### ISSUE TITLE
Not working for me. / Support for ESR v52.9.0 

### ISSUE THREAD

**Issue by PenBach** [igorlogius/persistent-element-remover#9]
I'm using Firefox ESR v52.9.0 (32-bit).
I can't seem to get this addon to work at all.

I want to block "Category" elements on my ebay Summary page.
So I have the following rule
URL RegEx=
https:\/\/www\.ebay\.co\.uk.*
CSS Selector=
carousel__viewport
for div class="carousel__viewport"

Do I need quotes around anything?
I have tried various different elements on the page, but nothing gets removed.
What happens with something like this, where there's a space and hyphens: class="carousel m-carousel-pills"

Am I doing something wrong?
Please help.

**Comment by igorlogius** [comment]
I think your css selector is just missing the `dot` before the class name. So it should be: 

`.carousel__viewport`

FYI: 
The css selector gets directly feed into the document.querySelectorAll function 
ref.  https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll
So you can pretty easily test your selector in the development console by running

`document.querySelectorAll('.carousel__viewport')`

If the output of this is the element you want to remove, thatn you can be confident that the selector you used is correct. 

Hope this info helps.

**Comment by igorlogius** [comment]
> What happens with something like this, where there's a space and hyphens: class="carousel m-carousel-pills"

This would be 2 classes, which you can just specifiy like this concat (just mind the preceding dots) like this: `document.querySelector('.carousel.m-carousel-pills')

FYI: 
You can use firefoxes built-in development tools, to get some pretty good suggestions for CSS selectors/paths. Just open the inspector view and right click on the element ... under the copy entry you should find the option to copy some CSS selector/path ... wich might not always be completely ok to use, but should give you a good starting point.  Most of the time you'll might only need to reduce add expand it a bit.

**Comment by PenBach** [comment]
Thank you for such a quick reply.
Some excellent tips there. Really helpful.

Unfortunately, even though the CSS Selector seems to be correct when checked in the Console, the page still loads everything.
I'm pretty sure the URL is correct; I checked it in regex101 to be sure.

I tried a simpler CSS Selector
.global-header
and it still doesn't work.

Firefox ESR v52.9.0 (32-bit) is OK?
Any idea what I might be missing here?
Any FF permissions I might need to change?
